.lando.yml

```diff
@@ -0,0 +1,75 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install --no-scripts
```

Comment:
Why is this on `run` and not `build`?

Reply:
I agree that `build` is more appropriate. Since we are running Composer 2 on the container, and it is just an `install`, speed is not an issue. I can't think of any practical reason to care. Maybe if the scaffold plugin runs even with `--no-scripts` and you have some changes that you want to preserve?